src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService+UITestingMethods.swift

```diff
@@ -258,14 +282,51 @@ extension HealthCertificateService {
 				),
 				certificateToReissue: nil,
 				accompanyingCertificates: nil,
-				certificates: [ DCCReissuanceCertificateContainer(
-					certificateToReissue: DCCCertificateContainer(certificateRef: DCCCertificateReference(barcodeData: certifiedPerson.healthCertificates.last?.base45 ?? "")),
-					accompanyingCertificates: [],
-					action: "renew"
-				)]
+				certificates: [
+					DCCReissuanceCertificateContainer(
+						certificateToReissue: DCCCertificateContainer(
+							certificateRef: DCCCertificateReference(barcodeData: firstReissuanceBase45)
+						),
+						accompanyingCertificates: [
+							DCCCertificateContainer(
+								certificateRef: DCCCertificateReference(barcodeData: secondReissuanceBase45)
+							),
+							DCCCertificateContainer(
+								certificateRef: DCCCertificateReference(barcodeData: thirdReissuanceBase45)
+							),
+							DCCCertificateContainer(
+								certificateRef: DCCCertificateReference(barcodeData: forthReissuanceBase45)
+							)
+						],
+						action: "renew"
+					),
+					DCCReissuanceCertificateContainer(
+						certificateToReissue: DCCCertificateContainer(
+							certificateRef: DCCCertificateReference(barcodeData: fifthReissuanceBase45)
+						),
+						accompanyingCertificates: [],
+						action: "renew"
+					)
+				]
 			),
 			certificatesRevokedByInvalidationRules: dccWalletInfo.certificatesRevokedByInvalidationRules
 		)
 	}
+	private var firstReissuanceBase45: String {
+		"HC1:6BFOXN*TS0BI$ZDFRH5TSWK3*93X+01V80RBZEJ8USXG43NI 7EFXRV7P4XUS4FDMP7JM:UC*GP-S4FT5D75W9AV88E34L/5R3F9JAY-BI9EW+9K.CMIAO/B.OC4-9D.B6LFDC9 AE 2AB7LOKEH-BXPLV6I-TL8KES/F-1JZ.KBIH:HG43MWY8%FAGUU0QIRR97I2HOAXL92L0: KQMK8J4RK46YBBOAXM42%K:XFEK0OH6PO9:TUQJAJG9-*NIRICVELZUZM9EN9-O97LAQCK /P/T14SIWG9$R7064B69X5QNEQT56QHE+.6L80TM8$M8SNCNAK09TU*5KGIDPS7A8Q/DA-7TLOJ3KV:BW.7-$4UGGBEG*Z1W1O4WDQETN:523EDZT/P7+7S/ZEJZ9Q:NIBV84W1-HGN33SNVMAS6C4XL QQU6S9QLJ KX$FTDAM2H3TS3KNY%AI:OF/9NHB%3KKS06GBF E"
+	}
```

Comment:
Couldn't these just be `let`s?

Reply:
@nickguendling  'let' declarations cannot be computed properties